GVRf/Framework/jni/oculus/activity_jni.cpp

```diff
@@ -113,22 +113,11 @@ jclass GVRActivity::GetGlobalClassReference( const char * className ) const
     return gc;
 }
 
-void GVRActivity::ConfigureVrMode( ovrModeParms & modeParms )
+void GVRActivity::Configure( OVR::ovrSettings & settings )
 {
-    LOG( "GVRActivity::ConfigureVrMode");
-
-    modeParms.CpuLevel = 2;
-    modeParms.GpuLevel = 2;
-
-    // Always use 2x MSAA for now
-    app->GetVrParms().multisamples = 2;
-
-    // Always use bilinear texture filtering.
-    app->GetVrParms().textureFilter = TEXTURE_FILTER_BILINEAR;
-
-    app->GetSwapParms().WarpProgram = WP_SIMPLE;
+    LOG( "GVRActivity::Configure");
+    // leave it as the oculus defaults for now.
```

Comment:
what we _really_ should do is pay attention and pass down the value that is set in the xml file.  we should also expose some apis for setting the cpu/gpu levels, srgb framebuffers, etc. that would be part of ovrSettings in here.  That should be part of a separate check-in.  In the meantime, i'll take your advice and explicitly set multisample to 2 since that's what we already had hard-coded before.